src/page-layout/NoticesLayout/components/Filter/AddressSelector/index.ts

```diff
@@ -0,0 +1,3 @@
+import AddressSelector from "./AddressSelector";
+
+export default AddressSelector;
```

Comment:
린트 룰을 끄고 이렇게 표현해줘도 괜찮습니다~
```suggestion
export { default } from "./AddressSelector";
```